src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -110,9 +108,12 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
           }),
           physicalConstraintForm: this.fb.group({
             // Maximum slope allowed for planning area
-            maxSlope: ['', Validators.min(0)],
-            // Maximum road distance
-            maxRoadDistance: ['', Validators.min(0)],
+            maxSlope: ['', 
+              [Validators.min(0), Validators.max(100), Validators.required]],
+            // Minimum distance from road allowed for planning area
+            // TODO update variable name to minDistanceFromRoad 
```

Comment:
Seems like this is an easy TODO to tackle now vs later, not sure if this change is coming up next on a follow up pr?

Reply:
Haha thank you for asking that so I can give context, that TODO felt silly to put down on its own. I thought I could just add that change to this pr but as I went through all the references I realized it expands through several components and services. In the interest of keeping these pr's slim and easy to review, I opted to put that change in the next pr.